src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,29 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProfileServlet extends HttpServlet {
+	  @Override
```

Comment:
Nit: the alignment here looks weird in GitHub to me? You could check if there's maybe a tab instead of spaces or something like that.